modules/auxiliary/scanner/http/xst.rb

```diff
@@ -0,0 +1,64 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Auxiliary
+
+  # Exploit mixins should be called first
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::WmapScanServer
+  # Scanner mixin should be near last
+  include Msf::Auxiliary::Scanner
+
+  def initialize
+    super(
+      'Name'        => 'Cross-Site Tracing (XST) Checker',
+      'Description' => 'Checks if the host is vulnerable to Cross-Site Tracing (XST)',
+      'Author'       => [ 'Jay Turla <@shipcod3>' ],
+      'License'     => MSF_LICENSE
+    )
+  end
+
+  def run_host(target_host)
+
+    begin
+      res = send_request_raw({
+        'version'      => '1.0',
+        'uri'          => '/<script>alert("TRACE Payload");</script>', #XST Payload
+        'method'       => 'TRACE',
+        'headers' =>
+        {
+          'Cookie' => "TRACE ME PLS",
```

Comment:
@void-in I think that's just okay but feel free to change something though :)

@jvazquez-r7 not really, should I just remove it?